### ISSUE TITLE
The Regularizer does not work in combination with Fused batchnorm

### ISSUE THREAD

**Issue by Timen** [rothk/Stabilizing_GANs#1]
Something I ran into while trying out the regularizer in my own application is that it just hangs during session creation/initialization. Not sure why something with gradient computation and using fused batch norm according to my traceback. 

It is however easily fixed by not using the fused batch norm function (in slim.batch_norm just set the fused parameter to False.). 

If somebody wants to look into it, my application uses a semi deep resnet based network, created with the slim library.
